src/instr/instr_paje_trace.cpp

```diff
@@ -218,23 +212,16 @@ void TRACE_paje_end() {
   XBT_DEBUG("Filename %s is closed", filename);
 }
 
-DefineContainerEvent::DefineContainerEvent(type_t type)
+void DefineContainerEvent(type_t type)
 {
-
-  event_type                            = PAJE_DefineContainerType;
-  timestamp                             = 0;
-  this->type = type;
+  e_event_type event_type                            = PAJE_DefineContainerType;
```

Comment:
I think that all event_type variables can be removed: you can use its value directly now (here: PAJE_DefineContainerType)